covigator/dashboard/figures/lineages.py

```diff
@@ -36,8 +37,20 @@ def get_lineages_plot(self, data_source: str, countries=None, lineages=None):
                 color_discrete_sequence=px.colors.qualitative.Vivid)
             fig1.update_traces(line=dict(width=0.5), showlegend=False)
 
+            # Perform smoothing by using a simple moving average
+            # If time_period is False the un-smoothed data is plotted
+            smooth_data = data[['date', 'lineage', 'ratio_per_date', 'cumsum', 'count']]
+            if time_period:
+                # Group data by lineage to calculate correct average for each lineage in df
+                sma_df = smooth_data.groupby('lineage')[['lineage', 'ratio_per_date', 'cumsum', 'count']]
+                sma_df = sma_df.rolling(time_period, min_periods=1).mean()
+                sma_df = sma_df.reset_index(level='lineage')[['ratio_per_date', 'cumsum', 'count']]
+                # Update columns with moving average over selected period
+                smooth_data.loc[:, 'ratio_per_date'] = sma_df['ratio_per_date']
+                smooth_data.loc[:, 'cumsum'] = sma_df['cumsum']
```

Comment:
I was wondering if we want to do the rolling average also on `cumsum` and `count` considering that these values are only shown in the hover data, hence they do not affect the looks of the plot. What do you think? I don't have a strong preference here, just wondering.